### DISCUSSION TITLE
How to capture image,video,or timelapse using web interface

### DISCUSSION THREAD

**Discussion by Whimick** [Lillifee/raspiCam#48] [Q&A]
Hi,
I have just came across you app, is there any way to capture a image,video,or timelapse using web interface, on firfox etc.

I have not so far found any capture buttons to the  web interface, if they not yet added, is there any chance of adding them.

Many Thanks Michael

**Comment by Lillifee** [comment]
Hey @Whimick ,

Yeah should be also possible on firefox.
You can capture an image or video using the round button:
![image](https://user-images.githubusercontent.com/23337626/227253538-9a9836be-5075-4951-b482-1260bfcd8f04.png)
